README.rst

```diff
@@ -1,5 +1,5 @@
 
-|LASSO| Python Library

```

Comment:
You want to remove the logo? Found it usually quite nice. We could also do a small poll on discord what the others think.